ethcontract-common/src/artifact.rs

```diff
@@ -204,26 +217,32 @@ mod test {
 
         assert_eq!(artifact.len(), 0);
 
-        let insert_res = artifact.insert(make_contract("C1"));
+        {
```

Comment:
Unfortunately those scopes are needed due to us needed to update a modified contract. (see `impl Drop for ContractMut<'_>`.